app/assets/javascripts/heatmap.js

```diff
@@ -59,9 +60,10 @@ document.addEventListener("DOMContentLoaded", () => {
       },
       scale: {
         color: {
-          type: "threshold",
+          type: "sqrt",
+          // reverse mutates the array. But with theme changes we must work on the original data.
```

Comment:
What does this comment say?

If we need to stop contributors from trying `rangeColors.reverse()` [again](https://github.com/openstreetmap/openstreetmap-website/pull/5402#discussion_r1923048595) and again, we can have two different `rangeColors` arrays defined earlier. Maybe the colors in them also need to be different, not just reversed.